samples/openthread/coprocessor/boards/nrf54lm20pdk_nrf54lm20a_cpuapp.conf

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_SPI_NOR=n
+
+# Workaround required as Zephyr L2 implies usage of NVS backend for settings.
+# It should be removed once the proper fix will be applied in Zephyr.
+CONFIG_NVS=n
+CONFIG_ZMS=y
+CONFIG_SETTINGS_ZMS=y
+
+# ZMS cache optimization
```

Comment:
This is maybe minor, but I'm wondering whether we need cache in openthread samples. There is some inconsistency in configuration as we have cache enabled for L15, but not for L10 and L05. In Matter we have it enabled for all L variants. I would expect we also have this enabled or not for all boards in a consistent way.

Reply:
OT samples rather do not store big amount of data in settings, so I would guess it may not be necessary to use cache.

Reply:
From the past we had seen issues with the bare OpenThread applications as well, so I would defiantly keep it. If L05 and L10 do not have cache enabled, I would add it.